### ISSUE TITLE
Unexpected end of JSON input when ic.RawState is empty

### ISSUE THREAD

**Issue by avarabyeu** [slack-go/slack#829]
UPD: The issue has been fixed in master branch, are there any estimated on when this can be released?

Example of interaction request:
```json
{
  "type": "block_actions",
  "user": {
    "id": "XXX",
    "username": "XXX",
    "name": "XXX",
    "team_id": "XXX"
  },
  "api_app_id": "XXX",
  "token": "XXX",
  "container": {
    "type": "message",
    "message_ts": "1602851656.005000",
    "channel_id": "XXX",
    "is_ephemeral": false
  },
  "trigger_id": "1457288895456.506668261542.c3d01837c251db0c168beef99548cf93",
  "team": {
    "id": "XXX",
    "domain": "XXX"
  },
  "channel": {
    "id": "XXX",
    "name": "directmessage"
  },
  "message": {
    "bot_id": "XXX",
    "type": "message",
    "text": "This content can't be displayed.",
    "user": "XXX",
    "ts": "1602851656.005000",
    "team": "XXX",
    "blocks": [
      {
        "type": "section",
        "block_id": "5bg",
        "text": {
          "type": "mrkdwn",
          "text": "You can add a button alongside text in your message",
          "verbatim": false
        }
      },
      {
        "type": "actions",
        "block_id": "hOue9",
        "elements": [
          {
            "type": "button",
            "action_id": "qa5=o",
            "text": {
              "type": "plain_text",
              "text": "Click me!",
              "emoji": true
            },
            "value": "the button just works"
          }
        ]
      }
    ]
  },
  "response_url": "https:\/\/hooks.slack.com\/actions\/XXX\/XXX\/XXX",
  "actions": [
    {
      "action_id": "qa5=o",
      "block_id": "hOue9",
      "text": {
        "type": "plain_text",
        "text": "Click me!",
        "emoji": true
      },
      "value": "the button just works",
      "type": "button",
      "action_ts": "1602851674.447323"
    }
  ]
}
```

**Comment by hanenao** [comment]
I've encountered the same phenomenon; I've contacted Slack for help.

**Comment by hanenao** [comment]
I've encountered the same phenomenon; I've contacted Slack for help.

**Comment by kanata2** [comment]
Released! Please use v0.7.2 for workaround of this behaviour.
https://github.com/slack-go/slack/releases/tag/v0.7.2

**Comment by kanata2** [comment]
@hanenao Thanks! I would appreciate if you could share the result of inquiry once you received 🙏
